cupy/core/raw.pyx

```diff
@@ -33,7 +33,7 @@ cdef class RawKernel:
     """
 
     def __init__(self, code, name, options=(), backend='nvrtc', *,
-                 translate_cucomplex=False):
+                 translate_cucomplex=False, grid_sync=False):
```

Comment:
I think grid synchronization is only a part of `cooperative_groups`. How about something like `enable_cooperative_groups`?

Reply:
Fixed.